ui/DesignSystem/Component/Header/Large.svelte

```diff
@@ -1,34 +1,29 @@
 <script lang="ts">
   import type { EmojiAvatar } from "../../../src/avatar";
-
-  import type { Identity } from "../../../src/identity";
-  import type { Org } from "../../../src/org";
+  import type { Stats } from "../../../src/project";
 
   import { Avatar, Icon } from "../../Primitive";
   import Urn from "../Urn.svelte";
 
   export let name: string;
   export let urn: string;
+  export let description: string | undefined = undefined;
```

Comment:
```suggestion
  export let description = "";
```

Reply:
bc it's an optional string prop ✨

Reply:
For me `optional` means, the `description` would be `string | undefined`. But if I understand correctly, with the above notation it is `string(set by user) | string(empty)`.
That means if I don't provide a description it is set to `""` and code relying on if description was provided has to do `{#if description.length > 0}` instead of `{#if description}`

Reply:
Correct, we discussed this [here](https://github.com/radicle-dev/radicle-upstream/pull/867#discussion_r482532377) and went with the empty string as a default value, which also ensures we handle the empty string state in UI (i.e. if the parent passes an empty string, we probably shouldn't render empty `div`s).

Reply:
9f0bdbd